GlyssenEngineTests/Quote/QuoteParserTests.cs

```diff
@@ -5370,6 +5370,28 @@ public void Parse_MultipleInterruptionsInMiddleOfMultiBlockQuote_MultiBlockQuote
 			Assert.AreEqual(CharacterVerse.kScriptureCharacter, output[5].CharacterId);
 		}
 		#endregion
+
+		[TestCase(19, 10, "6", "narrator-PSA")]
+		[TestCase(40, 7, "8", "Jesus")]
+		public void GetCharacters_ReferenceTextIsNull_ReturnsOnlyOneCharacterForVerse(int book, int chapter, string verse, string expectedCharacter)
+		{
+			var cvRepo = new ParserCharacterRepository(ControlCharacterVerseData.Singleton, null);
+
+			var character = cvRepo.GetCharacters(book, chapter, new Verse(verse));
```

Comment:
In both this test and the next, the variable should be named "characters" since it represents a collection (even if there only ends up being one character in it).